app/javascript/src/components/time-tracking/Index.tsx

```diff
@@ -56,6 +58,8 @@ const TimeTracking: React.FC<Iprops> = ({
   // sorting by client's name
   clients.sort((a: object, b: object) => a["name"].localeCompare(b["name"]));
 
+  const employeeOptions = employees.map(e => ({ value: `${e["id"]}`, label: e["first_name"] + " " + e["last_name"] }) );
+
```

Comment:
```suggestion
  const employeeOptions = employees.map(e => ({ value: e["id"], label: e["first_name"] + " " + e["last_name"] }) );

```
or `value: String(e["id"])` if the intention is to convert it into string.